eegdash/dataset.py

```diff
@@ -52,8 +45,11 @@ def __init__(
             "R2": "ds005506",
             "R1": "ds005505",
         }
+        query = additional_query or {}
+        assert "dataset" not in querry
+        querry["dataset"] = dsnumber_release_map[release] 
         super().__init__(
-            query={"dataset": dsnumber_release_map[release]},
+            query=query,
             cache_dir=cache_dir,
             s3_bucket=s3_bucket,
             **kwargs,
```

Comment:
We can reuse the `query` argument to keep it consistent with the base class? Something like this
```
        dataset = dsnumber_release_map[release] 
        if query is None:
            query = {"dataset": dataset}
        elif "dataset" not in query:
            query["dataset"] = dataset
        elif query["dataset"] != dataset:
            raise ValueError(
                f"Query dataset {query['dataset']} does not match the release {release} "
                f"which corresponds to dataset {dataset}."
            )
        
        super().__init__(
            query=query,
            cache_dir=cache_dir,
            s3_bucket=s3_bucket,
            **kwargs,
        )
```